app/controllers/submissions_controller.rb

```diff
@@ -22,12 +22,12 @@ def create
     @ontology.update
     
     @submission_saved = @submission.save
-    if !@submission_saved || @submission_saved.errors
+    if response_error?(@submission_saved)
```

Comment:
The `response_error?` method worked with other object types, but not for submissions. If a submission is successfully saved, the return value in `@submission_saved` is a `LinkedData::Client::Models::OntologySubmission` object, which has a `status` method attached to represent submission status. When `response_errors?` is subsequently called, I get:

```
ArgumentError (comparison of String with 400 failed):
  
app/controllers/application_controller.rb:198:in `<'
app/controllers/application_controller.rb:198:in `response_success?'
app/controllers/application_controller.rb:202:in `response_error?'
app/controllers/submissions_controller.rb:25:in `create'
```

The end result is that the new submission is created, but the page shows a 500 error.

Reply:
fixed